manager/manager.go

```diff
@@ -640,6 +643,27 @@ func (m *manager) Exists(containerName string) bool {
 	return false
 }
 
+func (m *manager) GetProcessList(containerName string, options v2.RequestOptions) ([]v2.ProcessInfo, error) {
+	// override recursive. Only support single container listing.
+	options.Recursive = false
+	conts, err := m.getRequestedContainers(containerName, options)
+	if err != nil {
+		return nil, err
+	}
+	if len(conts) != 1 {
+		return nil, fmt.Errorf("Expected the request to match only one container")
+	}
+	// TODO(rjnagal): handle count? Only if we can do count by type (eg. top 5 cpu users)
+	ps := []v2.ProcessInfo{}
+	for _, cont := range conts {
```

Comment:
loop was simpler because conts is a map and containerName isn't necessarily the key.